core/frontend/src/test/setupTests.ts

```diff
@@ -1,7 +1,6 @@
 // By importing a barrel file within a setup file, we would be disabling vitest's ability to mock modules. But it's the easiest way to avoid circular import runtime errors within vitest.
 // Link to a section covering this issue: https://vitest.dev/guide/common-errors.html#cannot-mock-mocked-file-js-because-it-is-already-loaded
 // Not importing all of core frontend, as it's not needed. Only MockRender namespace is causing circular import runtime errors.
-import "../internal/render/MockRender";
-
+import "../core-frontend";
```

Comment:
The comment above states "not importing all of core frontend" yet...

Reply:
okay, updated comment, we have to import all of core-frontend